keps/sig-node/4112-passdown-resources-to-cri/README.md

```diff
@@ -0,0 +1,1277 @@
+
+Consider the following in developing a version skew strategy for this
+enhancement:
+- Does this enhancement involve coordinating behavior in the control plane and
+  in the kubelet? How does an n-2 kubelet without this feature available behave
+  when this feature is used?
+- Will any other components on the node change? For example, changes to CSI,
+  CRI or CNI may require updating that component before the kubelet.
+-->
+
+The feature is node-local (kubelet-only) so there is no dependencies or effects
+to other Kubernetes components.
+
+The behavior is unchanged if either kubelet or the CRI runtime running on a
+node does not support the feature. If kubelet has the feature enabled but the
+CRI runtime does not support it, the CRI runtime will ignore the new fields in
+the CRI API and function as previously. Similarly, if the CRI runtime supports
+the feature but the kubelet does not, the runtime will resort to the previous
+behavior.
+
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable; can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+<!--
+Pick one of these and delete the rest.
+
+Documentation is available on [feature gate lifecycle] and expectations, as
+well as the [existing list] of feature gates.
+
+[feature gate lifecycle]: https://git.k8s.io/community/contributors/devel/sig-architecture/feature-gates.md
+[existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
+-->
+
+- [X] Feature gate
+  - Feature gate name: KubeletContainerResourcesInPodSandbox
+  - Components depending on the feature gate:
+    - kubelet
+
+###### Does enabling the feature change any default behavior?
+
+<!--
+Any change of default behavior may be surprising to users or break existing
+automations, so be extremely careful here.
+-->
+
+The default behavior in Kubernetes is unchanged.
```

Comment:
Well, not exactly. It changes the behavior of the kubelet for every pod on that node, in that it now passes the information to the runtime. Then your next paragraph makes more sense if it says "This may cause change in behavior...".

Reply:
Thanks, I see this was contradictory :face_with_diagonal_mouth: What I tried to express is that enabling this in Kubernetes alone wouldn't change any default behavior...

I removed the first paragraph entirely. This section now states:

```
There might be changes in behavior if the underlying CRI runtime depends...
```

PTAL @johnbelamaric

Reply:
Closing as resolved